worker/src/queues/repeatQueue.ts

```diff
@@ -1,27 +0,0 @@
-import { Processor, Queue } from "bullmq";
-
-import { redis, QueueJobs, QueueName } from "@langfuse/shared/src/server";
-import { enqueueBatchExportJobs } from "../features/batchExport/enqueueBatchExportJobs";
-
-export const repeatQueue = redis
-  ? new Queue(QueueName.RepeatQueue, {
-      connection: redis,
-      defaultJobOptions: { removeOnComplete: true, removeOnFail: true },
-    })
-  : null;
-
-if (repeatQueue) {
-  repeatQueue.add(
-    QueueJobs.EnqueueBatchExportJobs,
-    {},
-    {
-      repeat: { pattern: "*/10 * * * *" },
-    },
-  );
-}
```

Comment:
Initially that message was sent via HTTP to the worker, we had this periodic check added additionally to check the DB state. 

Since we are writing into the queue directly from `web` [here](https://github.com/langfuse/langfuse/blob/8b686c9ecab7f744bfc099f8e311a0b8acd03171/web/src/server/api/routers/batchExport.ts#L76), this is now indeed unnecessary